iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/env/RegionMaintainHandler.java

```diff
@@ -667,12 +667,21 @@ public void removeDataNodePersistence(TDataNodeLocation dataNodeLocation) {
    */
   public void transferRegionLeader(TConsensusGroupId regionId, TDataNodeLocation originalDataNode)
       throws ProcedureException, InterruptedException {
+    List<TDataNodeLocation> excludeDataNode = Collections.singletonList(originalDataNode);
+    transferRegionLeader(regionId, originalDataNode, excludeDataNode);
+  }
+
+  public void transferRegionLeader(
+      TConsensusGroupId regionId,
+      TDataNodeLocation originalDataNode,
+      List<TDataNodeLocation> excludeDataNode)
```

Comment:
seems we only need one node which will be deleted in the future? not a list?

Reply:
It's convenient to use to determine whether an object is included